linchpin/utils/dataparser.py

```diff
@@ -52,6 +52,9 @@ def process(self, file_w_path, data='{}', data_w_path=None):
             An optional run_id if the task is idempotent or a destroy action
         """
 
+        if not data:
+            data = '{}'
```

Comment:
why would this be needed if line 39 has data='{}' ?

Reply:
Because other locations pass in None for this and it would require reworking those functions as well to ensure the json dict representation was passed in. Instead, I just adjusted here. However, I will also adjust line 39 to be data=None.